### DISCUSSION TITLE
Whether to consider supporting multi-team switching in the future?

### DISCUSSION THREAD

**Discussion by jinrenjie** [outline/outline#3919] [Self Hosting] [answered]
At present, Outline's document organization structure and permission assignment cannot adapt to the multi-team collaboration and sharing scenarios. 

Will you consider opening the function of multi-team switching in the future?

**Comment by tommoor** [comment, marked as answer]
We are working on the foundations for this feature right now. It will be available in the cloud and enterprise editions.

**Reply by jinrenjie** [reply]
Thank you for your reply!
